### PR TITLE
#2560 fix(cli): add better keptn upgrade message

### LINKED ISSUE #2560: Better instructions on how to download new version of CLI to do a keptn upgrade
As the Keptn team is constantly putting out new cool features and with that also an updated version of the CLI which is needed to upgrade to the latest Keptn I suggest some user experience improvements in that process.

#1: keptn version should give me the command to update my CLI
Currently when doing keptn version I get this
```console
Keptn CLI version: 0.7.1

keptn version 0.7.2 is available! Please visit https://keptn.sh for more information.
Daily version check is enabled.
Keptn will collect statistical data and will notify about new versions and security patches for Keptn. Details can be found at: https://keptn.sh/docs/0.7.x/reference/version_check
---------------------------------------------------
* To disable the daily version check, please execute:
 - keptn set config AutomaticVersionCheck false
```

I suggest to provide an easy - here is how you upgrade your CLI command. So - instead of just saying "please visit the keptn website" - I suggest to also add the following:
```
To update to the latest CLI run this command: curl -sL https://get.keptn.sh | sudo -E bash
```
Another option would be to point the user to a site that shows the user how to download the latest CLI depending on the target OS!

#2: Do a version check with keptn upgrade
Right now when doing a keptn upgrade and the CLI version is not on the latest available keptn version we get the following error message which is not that descriptive to the end user on what to do next:
```console
$ keptn upgrade
Error: No upgrade path exists from Keptn version 0.7.1 to 0.7.1
```

I suggest that keptn upgrade should also highlight that there is a new keptn version available but first the user has to upgrade to the latest CLI in order to perform that updgrade

### PR DESCRIPTION
**Pull request by DarthBenro008**
- Fixes #2560 

### Preview
keptn_version
keptn_upgrade

### CODE REVIEW

File: cli/pkg/version/version_checker.go [hunk 1]
```diff
@@ -129,7 +129,9 @@ func (v *VersionChecker) getNewerCLIVersion(usedVersionString string) (available
 	return res, nil
 }
 
-const newCompatibleVersionMsg = `* Keptn CLI version %s is available! Please visit https://keptn.sh/docs/%s/operate/upgrade/ for more information.`
+const newCompatibleVersionMsg = `* Keptn CLI version %s is available! 
+ - To update to the latest CLI run this command: curl -sL https://get.keptn.sh | sudo -E bash
```
Comment (christian-kreuzberger-dtx) [inline cli/pkg/version/version_checker.go:133]: Please use the `curl ...` command without `sudo -E`, as shown in our [Quickstart](https://keptn.sh/docs/quickstart/#2-1-install-keptn-cli).

Comment (DarthBenro008) [reply cli/pkg/version/version_checker.go]: @christian-kreuzberger-dtx my bad, I presumed sudo would be required to mv the binary to /usr/local/bin, I have fixed it!

Comment (christian-kreuzberger-dtx) [reply cli/pkg/version/version_checker.go]: It is, but we moved it inside the get.keptn.sh script ;) thanks for the fix

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**